aleemer/Internify
# created json file full of student objects from survey

#### User Story
IN-56 (Populate DB with Student profiles from survey results)

#### Changes made
A new directory named "Populations" has been created under the existing src -> assets directory
Inside this new directory, a file named "students.json" has been created; within this file there are now 62 student objects.  These students may populate the MongoDB database.

#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
Please look inside the file, carefully at the student objects, to confirm their structure.  Note, that the student schemas deviate from the JIRA ticket in the following ways:

1. Jira Ticket: codingLanguages: [string]! **------** student object, coding language is a key:value pair
2. Jira Ticket: frameworks: [string]!**------** student object, frameworks is a key:value pair
3. Jira Ticket: workTools: [string]!**------** student object, workTools is a key:value pair
4. Jira Ticket: degree: [string]**------** student object, degree is a key:value pair

These differences have been discussed and confirmed with @tommytwm, who created the Jira Ticket.  The thinking behind the Jira Ticket's use of [string] was to reduce the workload/ complications involved in creating the student objects; however, it is understood that a key: value pair for the items above is acceptable and generally more functional. 

Also note: though the "students.json" file currently has 62 student objects, an arbitrary number of students can be synthetically created, if need be.
